lighthouse-core/lib/lh-error.js

```diff
@@ -360,6 +360,12 @@ const ERRORS = {
     lhrRuntimeError: true,
   },
 
+  // Layout Shift trace events are found but without data
```

Comment:
```suggestion
  /** Layout Shift trace events are found but without data */
```
to get the on-hover pickup